### DISCUSSION TITLE
How to make AccordionItem accept JSX children

### DISCUSSION THREAD

**Discussion by stefanprobst** [adobe/react-spectrum#2208] [Q&A] [answered]
i am trying out the accordion alpha, and it's not clear to me how i can create accordion items with JSX children (plain text works fine), e.g.:

```jsx
<Accordion>
  <Item key="first" title="First item">
    <p>This is the first item with <a href="#">a link</a>.</p>
  </Item>
</Accordion>
```

it seems i can make it work by setting `hasChildItems={false}` on the `Item`, but that's probably not how this is supposed to work.

See [here](https://github.com/stefanprobst/react-spectrum/commit/bca2898f328549f151748a4c34d3dd498b907336) for a failing story (getting this error when toggling the accordion panel: `Unknown element <p> in collection.`)

**Comment by snowystinger** [comment, marked as answer]
We don't support this yet, but have an issue for it https://github.com/adobe/react-spectrum/issues/1989
